src/librustc/infer/canonical/mod.rs

```diff
@@ -104,7 +137,25 @@ pub enum CanonicalVarKind {
     Ty(CanonicalTyVarKind),
 
     /// Region variable `'?R`.
-    Region,
+    Region(ty::UniverseIndex),
+
+    /// Region variable `'?R`.
```

Comment:
I don't think you wanted the exact same comment as the one above?

Reply:
Fixed.